pkg/cli/cli.go

```diff
@@ -400,30 +401,268 @@ func (c *CLIRunner) GetMemberLags(
 
 // GetPartitions fetches the details of each partition in a topic and prints out a summary for
 // user inspection.
-func (c *CLIRunner) GetPartitions(ctx context.Context, topic string) error {
+func (c *CLIRunner) GetPartitions(
```

Comment:
Your changes to this file follow different coding styles than the rest of the file. Some things that stand out:
1. You are using raw logging instead of the `printer` on the cliRunner
2. You have introduced many public functions in this cli file that aren't actually CLI commands but helper commands for other CLI functions. Typically these below in the admin package in zkclient.go and brokerclient.go. It seems like the only public function we should add in this file is `GetPartitions`. Another benefit of doing things this way is we have a lot of automated testing facilities in the admin package. Moving more of this logic there will improve maintainability of topicctl

Reply:
For 1, will use c.printer()

Reply:
For 1. I used c.printer instead of logging

For 2. I moved all public functions to appropriate locations. If it looks good, please resolve this